### ISSUE TITLE
Matcher that accepts array instead of a single value

### ISSUE THREAD

**Issue by experimatt** [rmm5t/strip_attributes#51]
I just came across this gem and its fabulous. It's both intuitive and easy to use. The only thing I found myself wanting was a matcher that accepted an array instead of a single value. I threw together a quick hacky solution in my tests that works, but adds one to the total number of test cases for every element/field in the array:

```
  [:first_name, :last_name, :title].each do |attr|
    it { is_expected.to strip_attribute(attr).collapse_spaces }
  end
```

I'd propose either adding a separate `strip_attributes` matcher that takes an array of values as a parameter, or modifying the existing `strip_attribute` matcher to conditionally check if an array was passed.

I'd be happy to submit a PR for this, but I'd like to get someone else's approval & opinion on it before proceeding.

Thanks,

Matt

**Comment by rmm5t** [comment]
I would accept a PR that takes a list of arguments. Aliasing strip_attribute to strip_attributes would be cool too.

**Comment by rmm5t** [comment]
Closed by #52
